packages/graph/hash_graph/lib/graph/src/ontology/entity.rs

```diff
@@ -0,0 +1,178 @@
+use std::{borrow::Cow, fmt};
+
+use error_stack::{IntoReport, Report};
+use serde::{
+    de::{self, Deserializer, SeqAccess, Visitor},
+    Deserialize,
+};
+
+use crate::{
+    knowledge::Entity,
+    ontology::{entity_type::EntityTypeQueryPathVisitor, EntityTypeQueryPath},
+    store::query::{Path, QueryRecord},
+};
+
+#[derive(Debug, PartialEq, Eq)]
+pub enum EntityQueryPath<'q> {
+    Id,
+    OwnedById,
```

Comment:
Are we expecting ot add `created_by`, `updated_by` and `removed_by` as possible query paths here?